alphabets/German.txt

```diff
@@ -32,5 +32,6 @@ z
 ä
 ö
 ü
+ß
```

Comment:
My German isn't the best, but isn't this the same as a 'ss'? I think why it's missing is because you're supposed to normalise all ß -> ss